content/tickets.rst

```diff
@@ -136,15 +138,42 @@ Below is a list of used tags:
   - These are tasks that don’t require knowledge of the product or operating system 'know-how' in advance.
 
 
+Ticket components
+-----------------
+
+Components and their explanations:
+
+ * client-commons
+ * client-ftp
+ * client-http
+ * client-ssh
+ * component not set
+ * gateway
+ * infrastructure
+ * libs
+ * manager
+ * pr (I think this is website/marketing related?)
+ * server-commons
+ * server-ftp
+ * server-http
+ * server-ssh
+ * support (also includes documentation, anything that involves information to the customer)
+ * webadmin
+
+
 Milestone planning
 ==================
 
-At the start of each cycle, we create a milestone or a sprint that represents our focus for the next month or week. We add new tickets to the new milestone or move tickets from 'Horse's Easter' milestone.
+At the start of each cycle, we create a milestone that represents our focus for the next month or week. We add new tickets to the new milestone or move tickets from 'Horse's Easter' milestone.
```

Comment:
we no longer have "Horse's Easter`

I think that is better to do these changes on a separate ticket, and talk about how we manage the milestones...what is the non-scheduled-yet... and how to chose a milestone